app/views/users/show.html.erb

```diff
@@ -1 +1,16 @@
-<h1>Here is the user</h1>
\ No newline at end of file
+<h1>Here is the user</h1>
+<%= render @user %>
+
+<div class = "bio">
+  <h3>Bio</h3>
+  <%= @user.Bio %> 
+</div>
+
+
+<div class="short-posts">
+  <% @user.recent_3_posts.each do |post| %>
```

Comment:
You used `#recent_post` method to render the user's 3 most recent posts 🐐 Well done!